components/Layout/Layout.module.scss

```diff
@@ -0,0 +1,13 @@
+.container {
+    padding: 0 2rem;
+}
```

Comment:
You might want to consider adding a max width and margin to this container 
```suggestion
.container {
    padding: 0 2rem;
    max-width: 1440px; // or $site-width;
    margin: 0 auto;
}
```

This means that the footer and footer cta don't get split up too far.

![image](https://user-images.githubusercontent.com/18164832/155694654-6539fb18-11b1-459f-a5cb-d404b95533dd.png)

Reply:
Added.